tldw_Server_API/app/main.py

```
@@ -1793,6 +1793,22 @@ async def lifespan(app: FastAPI):
+        try:
+            import sentry_sdk
+            sentry_sdk.init(
+                dsn=_sentry_dsn,
+                traces_sample_rate=float(os.getenv("SENTRY_TRACES_SAMPLE_RATE", "0.1")),
+                environment=os.getenv("DEPLOYMENT_ENV", "development"),
+                release=os.getenv("OTEL_SERVICE_VERSION", "1.0.0"),
+                send_default_pii=False,
+            )
+            logger.info("App Startup: Sentry error tracking initialized")
+        except _STARTUP_GUARD_EXCEPTIONS as _sentry_err:
+            logger.warning(f"App Startup: Sentry initialization failed: {_sentry_err}")
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)